server/impl/src/main/resources/com/walmartlabs/concord/server/org/triggers/concord.yml

```diff
@@ -6,7 +6,7 @@ flows:
   - log: "Refreshing repository... ${event}"
   - task: repositoryRefresh
     in:
-      repositoryInfo: ${event.repositoryInfo}
+      repositoryInfo: ${event.repositoryInfo.stream().filter(r -> r.enabled).toList()}
```

Comment:
https://github.com/walmartlabs/concord/blob/master/plugins/tasks/concord/src/main/java/com/walmartlabs/concord/client/RepositoryRefreshTaskParams.java#L45

throws exception for empty repo list :)

Reply:
Ooooh good catch